ui/src/lib/gql/convert.ts

```diff
@@ -164,6 +175,52 @@ export const toAsset = (asset: AssetFragment): Asset => ({
     : "pending",
 });
 
+export const toCmsProject = (cmsProject: CmsProjectFragment): CmsProject => ({
+  id: cmsProject.id,
+  workspaceId: cmsProject.workspaceId,
+  name: cmsProject.name,
+  alias: cmsProject.alias,
+  description: cmsProject.description ?? null,
+  license: cmsProject.license ?? null,
+  readme: cmsProject.readme ?? null,
```

Comment:
With your type update to match the schema, you shouldn't need these fallback nulls anymore, so please update all converting functions appropriately to match types 🙏🏻